cmake/FindLibUpnp.cmake

```diff
@@ -8,7 +8,7 @@
 #  UPNP_HAS_REUSEADDR - If LinUPnP was built with SO_REUSEADDR support
 
 find_package(PkgConfig QUIET)
-pkg_check_modules (PC_UPNP QUIET libupnp-1.8 libupnp)
+pkg_check_modules (PC_UPNP QUIET libupnp)
```

Comment:
My bad, this should have been `pkg_search_module` instead

Reply:
OK, I'll update the PR.